docs/asciidoc/modules/ROOT/pages/import/web-apis.adoc

```diff
@@ -40,67 +43,36 @@ CALL apoc.load.csv("pathToCompressedFile/file.zip!pathToCsvFileInZip/fileName.cs
 
-To use the HDFS protocol we need to download and copy the additional jars not included in the APOC Library.
-We can download it from https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/{apoc-release}/apoc-redis-dependencies-{apoc-release}.jar[this link] or locally downloading the apoc repository:
-----
-git clone http://github.com/neo4j-contrib/neo4j-apoc-procedures
-cd neo4j-apoc-procedures/extra-dependencies
-./gradlew shadow
-----
-and a jar named `apoc-hadoop-dependencies-{apoc-release}.jar` will be created into the `neo4j-apoc-procedures/extra-dependencies/hadoop/build/lib` folder.
-
-Once that file is downloaded/created, it should be placed in the `plugins` directory and the Neo4j Server restarted.
-
-== Using Google Cloud Storage
-
-In order to use https://cloud.google.com/storage/[Google Cloud Storage], you need to add the following Google Cloud dependencies in the plugins directory:
-
-* api-common-1.8.1.jar
-* failureaccess-1.0.1.jar
-* gax-1.48.1.jar
-* gax-httpjson-0.65.1.jar
-* google-api-client-1.30.2.jar
-* google-api-services-storage-v1-rev20190624-1.30.1.jar
-* google-auth-library-credentials-0.17.1.jar
-* google-auth-library-oauth2-http-0.17.1.jar
-* google-cloud-core-1.90.0.jar
-* google-cloud-core-http-1.90.0.jar
-* google-cloud-storage-1.90.0.jar
-* google-http-client-1.31.0.jar
-* google-http-client-appengine-1.31.0.jar
-* google-http-client-jackson2-1.31.0.jar
-* google-oauth-client-1.30.1.jar
-* grpc-context-1.19.0.jar
-* guava-28.0-android.jar
-* opencensus-api-0.21.0.jar
-* opencensus-contrib-http-util-0.21.0.jar
-* proto-google-common-protos-1.16.0.jar
-* proto-google-iam-v1-0.12.0.jar
-* protobuf-java-3.9.1.jar
-* protobuf-java-util-3.9.1.jar
-* threetenbp-1.3.3.jar
-
-We've prepared an uber-jar that contains the above dependencies in a single file in order simplify the process. You can http://example-data.neo4j.org/apoc/google-cloud-storage-dependencies-3.5-apoc.jar[download it from here] and copy it to your plugins directory.
-
-You can use Google Cloud storage via the following url format:
+[cols="5,5"]
+|===
+| Protocol | Needed extra dependency
+| S3       | https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/{apoc-release}/apoc-aws-dependencies-{apoc-release}.jar[apoc-aws-dependencies-{apoc-release}.jar]
+| GCS      | https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/{apoc-release}/apoc-gcs-dependencies-{apoc-release}.jar[apoc-gcs-dependencies-{apoc-release}.jar]
+| Hadoop   | https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases/download/{apoc-release}/apoc-hadoop-dependencies-{apoc-release}.jar[apoc-hadoop-dependencies-{apoc-release}.jar]
+|===
+
+After copying the jars into the plugins directory, the database will need to be restarted.
+
+=== Using S3 protocol
+
+include::partial$enableS3Export.adoc[]
```

Comment:
The include here will contain expressions like "By default exporting to S3 is disabled", but this page is about import and not export. I think you need to split enableS3Export.adoc differently so that some information is different in import/web-apis and export/web-apis, while some is shared.

Reply:
This was not the page I wanted to import. There's nothing you need to do to enable loading from s3 once you have the dependency